### ISSUE TITLE
Add reporter

### ISSUE THREAD

**Issue by sorosz89** [sorosz89/api-test-seed#1]
Configure a simple test reporting. XML, HTML

**Comment by kgabor95** [comment]
Hi @sorosz89 
Should I add both XML and HTML reporters?

**Comment by sorosz89** [comment]
Yes, please! Let's create a PoC with multiple reporters: [configuring reporters](https://jestjs.io/docs/configuration#reporters-arraymodulename--modulename-options)

You can even check the GitHub Actions Reporter [click](https://jestjs.io/docs/configuration#github-actions-reporter)

Please proceed with the research and recommend! Thank you!

**Comment by sorosz89** [comment]
Well, GitHub Actions is not configured yet. So, the is a great issue candidate. 😀
